apps/src/componentLibrary/toggle/Toggle.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classnames from 'classnames';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './toggle.module.scss';
+
+interface ToggleProps {
+  checked: boolean;
+  onChange: () => void;
+  name: string;
+  value?: string;
+  label?: string;
+  disabled?: boolean;
+  position?: 'left' | 'right';
+  size?: 'xs' | 's' | 'm' | 'l';
+}
+
+const Toggle: React.FunctionComponent<ToggleProps> = ({
+  checked,
+  onChange,
+  name,
+  value,
+  label,
+  disabled = false,
+  position = 'left',
+  size = 'm',
+}) => {
+  return (
+    <label
+      className={classnames(
+        moduleStyles.toggle,
+        moduleStyles[`toggle-${size}`],
+        position === 'right' && moduleStyles['toggle-right']
+      )}
+    >
+      <div>
+        <input
+          type="checkbox"
+          name={name}
+          value={value}
+          checked={checked}
```

Comment:
I'm noticing that the screenreader isn't consistently reading the changed state. 

https://github.com/code-dot-org/code-dot-org/assets/9142121/89fa9e65-3b29-486c-944b-e57891ef1be6

I'm not sure how to fix this, and I don't even see the `value` or `checked` attribute coming through on the DOM. You are using the "correct" HTML structure as far as I can tell. Might this behavior have to do with not having a controlled component in Storybook, but rather hard-coding the props without onChange behavior?

Reply:
What screen reader are you using?
I've tested with https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related and everything works fine so I think it might also be a screen reader problem.
`Value` won't be readable in screen reader even if you pass it inside.
Also tested with `checked` and `onChange` props locally, worked the same way as without them. (toggle styles won't change until checkbox checked state'll change)

Reply:
the chrome screenreader is unfortunately very different from most others 😞. if you work from windows, NVDA has a full-featured free version; on mac, VoiceOver is the most common. not sure what's most common on linux.

Reply:
I'm using VoiceOver with Chrome, but I probably should be using VoiceOver with Safari –– I'll check that. You're clearly using the best practices, and some of it reads great, so I'm not super concerned about it.